.clang-tidy

```diff
@@ -22,30 +21,28 @@ CheckOptions:
     value:           '2'
   - key:             readability-braces-around-statements.ShortStatementLines
     value:           '2'
+  # type names
   - key:             readability-identifier-naming.ClassCase
     value:           CamelCase
   - key:             readability-identifier-naming.EnumCase
     value:           CamelCase
-  - key:             readability-identifier-naming.EnumConstantCase
-    value:           UPPER_CASE
-  - key:             readability-identifier-naming.MethodCase
-    value:           camelBack
   - key:             readability-identifier-naming.UnionCase
     value:           CamelCase
+  # method names
+  - key:             readability-identifier-naming.MethodCase
+    value:           camelBack
+  # variable names
   - key:             readability-identifier-naming.VariableCase
     value:           lower_case
-  - key:             readability-identifier-naming.ConstantCase
+  - key:             readability-identifier-naming.ClassMemberSuffix
+    value:           '_'
+  # const static or global variables are UPPER_CASE
```

Comment:
Thanks for adding the comments in this file - it makes it a lot easier to understand what's going on!

Reply:
Could you move this constant down lower though? It refers to lines 42-47, right?

Reply:
Sorry, typo on my part. Meant to say "this comment" - the comment on line 39 is describing lines 42-47, right?